### DISCUSSION TITLE
cesium-sensor-volumes.js  引入识别不了Cesium

### DISCUSSION THREAD

**Discussion by YinMuHuaDao** [zouyaoji/vue-cesium#591] [Q&A]
![微信图片_20240802151448](https://github.com/user-attachments/assets/7d39b284-a81e-4694-bbd7-925515f9a35b)
![微信图片_20240802151454](https://github.com/user-attachments/assets/35a56b2b-8dad-43fd-b46d-754a56d7ed9b)
![微信图片_20240802151549](https://github.com/user-attachments/assets/c3fda557-6b96-466b-817c-fd5cf476913f)

[vue-cesium-vite-starter-master.zip](https://github.com/user-attachments/files/16466930/vue-cesium-vite-starter-master.zip)

**Comment by zouyaoji** [comment]
你这个看起来是没引入css啊 在index.html中引入

<link rel="stylesheet" href="/Cesium/Widgets/widgets.css">

**Reply by YinMuHuaDao** [reply]
我在main.ts 里面引用了vue-cesium/dist/index.css   ， 按照您的说法我也在index.html 里面引用了好像也没用 。
另外我把 index.html 里 <script src="/Cesium/Cesium.js"></script>  <script src="/cesium-sensor-volumes.js"></script> 这两句删了之后能正常运行 。

**Reply by zouyaoji** [reply]
给的压缩包 怎么重现报错问题?

**Comment by YinMuHuaDao** [comment]
install 之后 再 run dev ，运行不了的话可能是我node版本太新了，没啥太麻烦的话就算了我就是研究下 ，还是感谢你抽出时间回复谢谢了。

**Comment by germination-grows** [comment]
![image](https://github.com/user-attachments/assets/088a6b11-5646-433b-b348-bb782b5fa61f)

**Reply by germination-grows** [reply]
还得引用一下 Cesium.js
